Makefile

```diff
@@ -168,6 +168,7 @@ test-e2e: ginkgo generated_files
 #  INSTANCE_METADATA: For REMOTE=true and running on GCE only.
 #  GUBERNATOR: For REMOTE=true only. Produce link to Gubernator to view logs.
 #	 Defaults to false.
+#  PARALLELISM: The number of nodes to run.  Defaults to 8
```

Comment:
It's not only meaningful when tied to a cloud provider, currently, but maybe that's a bug?